controller/cache/cache.go

```diff
@@ -239,7 +239,7 @@ func (c *liveStateCache) getCluster(server string) (clustercache.ClusterCache, e
 			res := &ResourceInfo{}
 			populateNodeInfo(un, res)
 			res.Health, _ = health.GetResourceHealth(un, cacheSettings.clusterSettings.ResourceHealthOverride)
-			appName := kube.GetAppInstanceLabel(un, cacheSettings.appInstanceLabelKey)
+			appName := kube.GetAppInstanceIdentifier(un, cacheSettings.appInstanceLabelKey)
```

Comment:
Please see comment in https://github.com/argoproj/gitops-engine/pull/121#discussion_r516370910


Instead of using `GetAppInstanceIdentifier` from gitops engine let's move it into Argo CD repo.